src/bundler.ts

```diff
@@ -282,6 +282,12 @@ export class Bundler {
     for (const bundle of bundles) {
       for (const exclude of this.excludes) {
         bundle.files.delete(exclude);
+        const excludeAsFolder = exclude.endsWith('/') ? exclude : exclude + '/';
```

Comment:
I mentioned URL only in one of the 3 places it was mentioned and the others said paths and files and subpaths -- So I've updated now, unifying the description as a URL.